### ISSUE TITLE
Feature request: Total Commander WDX plug-in support

### ISSUE THREAD

**Issue by Hunter23071985** [tablacus/TablacusExplorer#211]
Hello!
Do you have plans to make an add-on that allows to use Content plugins (WDX) to view tags and file information?

**Comment by tablacus** [comment]
It is impossible for now because I don't know how to implement a new column in the Explorer Browser control.

Regards,

**Comment by Hunter23071985** [comment]
You can display the information in the information window, not in the Explorer column.

**Comment by tablacus** [comment]
I am tied up.

**Comment by Hunter23071985** [comment]
Maybe [WDX Guide](https://totalcmd.net/plugring/wdx_guide.html) or [WDX Test](https://totalcmd.net/plugring/wdxtest.html) can help you to get something like [this](https://funkyimg.com/i/2TqUS.png) at the output?

**Comment by I-Vladimir** [comment]
I don't think it would be any column. It is probably a new content sensitive "File Details" panel. 
I can personally just right click on a file to get details. Using TC for about 20 years and never used that plugin.

**Comment by Hunter23071985** [comment]
**I-Vladimir**, I brought a screenshot of the "information window" in a previous post.
Probably more correct to call it "File Details" panel, so let it be, but the plugin is very necessary.
Many of us need to see specific parameters of video, photos and other files for their processing.
And this information does not get through the right click on the file.

**Comment by Hunter23071985** [comment]
**Gaku**, maybe [WinScript Advanced Content Plugin](https://www.ghisler.ch/board/viewtopic.php?t=44032) can help you?
